File: PRAK205-2210817210023-MuhammadShofyAkmal.py
import math
a = float(input("Tinggi        : "))
b = float(input("Sisi Miring   : "))
c=math.sqrt(b*b-a*a)
d=a+b+c
e=c*a/2
print ("Alas     = %.0f cm" % (c))
print ("Tinggi   = %.0f cm" % (a))
print ("Keliling = %.0f cm" % (d))
print ("Luas     = %.0f cm^2" % (e))

File: PRAK205-2210817210023-MuhammadShofyAkmal.c
#include <stdio.h>
#include <math.h>
int main () {
float a,b,c,d,e;
printf ("Tinggi        : ");  scanf ("%f",&a); 
printf ("Sisi Miring   : ");  scanf ("%f",&b);
c=sqrt(b*b-a*a);
d=a+b+c;
e=c*a/2;
printf ("Alas     = %.0f cm\n",c);
printf ("Tinggi   = %.0f cm\n",a);
printf ("Keliling = %.0f cm\n",d);
printf ("Luas     = %.0f cm^2\n",e);
}